Public/Src/Utilities/KeyValueStore/RocksDb/RocksDbStore.cs

```diff
@@ -148,14 +148,18 @@ private struct DefaultOptions
             /// Every time the RocksDb instance is open, the current log file is truncated, which means that if you
             /// open the DB more than once in a 12 hour period, you will only have partial information.
             /// </param>
+            /// <param name="openWriteOnly">
+            /// Have RocksDb open for writing only which enables us to bulk load.
+            /// </param>
             public RocksDbStore(
                 string storeDirectory,
                 bool defaultColumnKeyTracked = false,
                 IEnumerable<string> additionalColumns = null,
                 IEnumerable<string> additionalKeyTrackedColumns = null,
                 bool readOnly = false,
                 bool dropMismatchingColumns = false,
-                bool rotateLogs = false)
+                bool rotateLogs = false,
+                bool openWriteOnly = false)

```

Comment:
I wouldn't call this `openWriteOnly`, but `openBulkLoad` or similar (the db can be read when prepared for bulk load, so the name's misleading)